### DISCUSSION TITLE
Playerheads doenst show up in tablist

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#841] [Help] [answered]
Hi! my player head doesnt show up in the tablist, but i set a UuidProvider and my Skin is set correctly

**Comment by MrGazdag** [comment, marked as answer]
The client’s tablist only shows player heads when the server it is connected to authenticates with Mojang (online mode). This also works by using a proxy such as BungeeCord or Velocity with online mode enabled.

**Reply by DasLixou** [reply]
But how can i activate it with Minestom? I use this for UUID Provider:
`MinecraftServer.getConnectionManager().setUuidProvider(((playerConnection, username) -> fromTrimmed(MojangUtils.fromUsername(username).get("id").getAsString())));`

and:
```java
private static UUID fromTrimmed(String trimmedUuid) {
        StringBuilder builder = new StringBuilder(trimmedUuid.trim());
        try {
            builder.insert(20, "-");
            builder.insert(16, "-");
            builder.insert(12, "-");
            builder.insert(8, "-");
        } catch (StringIndexOutOfBoundsException e) {
            throw new IllegalArgumentException();
        }
        return UUID.fromString(builder.toString());
    }```

**Reply by MrGazdag** [reply]
You can enable online mode by calling `MojangAuth.init()` before starting the server.

**Reply by DasLixou** [reply]
Thanks :D
